test/modules/jupyter-widget/create-deck.spec.js

```diff
@@ -0,0 +1,48 @@
+// eslint-disable-next-line
+/* global document, window, global */
+import test from 'tape-catch';
+
+import {CompositeLayer} from '@deck.gl/core';
+import {ScatterplotLayer} from '@deck.gl/layers';
+
+class DemoCompositeLayer extends CompositeLayer {
+  renderLayers() {
+    return new ScatterplotLayer(this.props);
+  }
+}
+
+// Mock the loading of the script
+const TEST_LIBRARY_NAME = 'DemoLibrary';
+window[TEST_LIBRARY_NAME] = {DemoCompositeLayer};
+const oldFetch = window.fetch;
+window.fetch = url => {
```

Comment:
You need to apply the override right before you use it. This will break other tests.